### DISCUSSION TITLE
修复个人网站失效链接

### DISCUSSION THREAD

**Discussion by giscus** [liolok/liolok.com#10] [Comments]
# 修复个人网站失效链接

李皓奇

https://liolok.com/zhs/fix-broken-urls-of-my-site/

**Comment by zhullyb** [comment]
甚至「尽量永远活着来维护网站」哈哈哈，皓奇太可爱了

**Comment by WeepingDogel** [comment]
皓奇哥哥！

我也想换个域名...

**Comment by CyrusYip** [comment]
可以试试 Vercel 这样的网站部署服务，它自带URL重定向功能。

我博客的重定向规则：https://github.com/CyrusYip/cyrusyip-blog/blob/4aa321b0deaaf97dd3dd535a2b4153a96d0e558a/vercel.json#LL10C17-L10C17

我每次改完链接格式之后都要作重定向，因为很难知道自己的旧链接都在哪，也很难让别人改。还是自己做好吧。

**Comment by weearc** [comment]
好奇老师找的好工具。下次搞seo试试看
